.gitmodules

```diff
@@ -39,3 +39,7 @@
 	path = external/cecil-legacy
 	url = git://github.com/mono/cecil.git
 	branch = mono-legacy-0.9.5
+[submodule "external/boringssl"]
+	path = external/boringssl
+	url = git@github.com:mono/boringssl.git
```

Comment:
This must use `git://` instead of `git@` otherwise it won't be accessible to people without push access to that repo.